ember-cli-build.js

```diff
@@ -77,6 +77,8 @@ module.exports = function(defaults) {
         development: 'bower_components/bootstrap-treeview/src/js/bootstrap-treeview.js',
         production: 'bower_components/bootstrap-treeview/dist/js/bootstrap-treeview.min.js'
     });
+    app.import('bower_components/jquery.tagsinput/src/jquery.tagsinput.js');
```

Comment:
Sadly, this is necessary for now. 

It's not automatic because we didn't want every data model consumer to bloat their apps with our bower dependencies. Thanks for adding this. :)